requirements.txt

```diff
@@ -1,5 +1,10 @@
-
+anticipy==1.0.1
```

Comment:
to get this to load i had to change to `ocha-anticipy` and remove the `==` pin. Should be fine to adjust in more up to date branch?